package.json

```diff
@@ -42,6 +42,7 @@
     "govuk-elements-sass": "^3.1.2",
     "govuk-react-components": "^1.2.0",
     "govuk_frontend_toolkit": "^7.4.1",
```

Comment:
I'd probably expect `govuk-elements-sass` and `govuk_frontend_toolkit` to be removed from here.